### DISCUSSION TITLE
Is it possible to make a translation version of SF2?

### DISCUSSION THREAD

**Discussion by Rel-Oiv** [ShiningForceCentral/SF2DISASM#6] [Q&A] [answered]
Hello, I found this a few days ago and I find it very cool.
I have a question.
Is it possible to make a translation version of SF2 with additional letters (5 letters and their upper case, 10 in total)? I tried to find the file where the font code is, but I only found the font graphics, to see if I can add more letters.

Thanks for the hard work 💪

**Comment by TiMMyyMMiT** [comment, marked as answer]
This is a long dead question but I wanted to make sure that it has an answer.
The answer is: YES 😁 

As proof, someone is currently making a mod that has added a good number of custom characters.
<img width="578" height="540" alt="image" src="https://github.com/user-attachments/assets/4f4d89be-633e-49ef-a838-43a56ee19eac" />
The TextEditor has recently been improved to better support custom symbols/languages.
